applications/asset_tracker_v2/src/modules/Kconfig.modem_module

```diff
@@ -67,6 +67,21 @@ config MODEM_NEIGHBOR_CELLS_DATA_CONVERT_RSRQ_TO_DB
 	  Don't convert RSRQ to dB when building for nRF Cloud, this is handled during encoding
 	  using the nRF Cloud cellular positioning library.
 
+choice MODEM_NEIGHBOR_SEARCH_TYPE
+	prompt "Type of search for neighbor cell measurements"
+	default LTE_LC_NEIGHBOR_SEARCH_TYPE_DEFAULT
```

Comment:
Some help text would be helpful here. I think the users needs to be aware that searching with the extended types will take longer time than the default. You could also refer to the nRF9160 AT commands specification for users that wants a little bit more information on the feature.